ftl/core/deck-config.ftl

```diff
@@ -424,10 +424,14 @@ deck-config-a-100-day-interval =
         [one] A 100 day interval will become { $days } day.
        *[other] A 100 day interval will become { $days } days.
     }
-deck-config-percent-of-reviews =  
-    { $reviews ->
-        [one] { $pct }% of { $reviews } review
-       *[other] { $pct }% of { $reviews } reviews
+deck-config-percent-of-items =  
+    { $count ->
+        [one] { $pct }% of { $count } item
```

Comment:
Let's call them cards rather than items, because that's what they actually are. In hindsight, FSRSCard might have been a better name for them.

Reply:
No, ignore me. I always forget that we create multiple FSRSItems for a card.

Reply:
What exactly is an FSRS item, by the way? I think that it is better to document it somewhere because the users will begin asking it that after this is merged.

Reply:
This is how it's documented in fsrs-rs: https://github.com/open-spaced-repetition/fsrs-rs/blob/e34265a5de7a9fbb138ea8b600deae44cf05a129/src/dataset.rs#L16